widget/embedded/src/pages/ConfirmSwapPage.tsx

```diff
@@ -324,15 +324,15 @@ export function ConfirmSwapPage() {
           <Typography variant="title" size="small">
             {i18n.t('You get')}
           </Typography>
-          <Button
-            style={{ padding: '0' }}
-            variant="ghost"
-            disabled={fetchingConfirmationQuote}
-            onClick={onRefresh}>
-            <div className={iconStyles()}>
-              <RefreshIcon size={16} />
-            </div>
-          </Button>
+          <div className={iconStyles()}>
+            <RefreshButton
+              onClick={
+                !fetchingConfirmationQuote && !showWallets
+                  ? onRefresh
+                  : undefined
```

Comment:
It's better to deactivate auto refresh when any type of modal is open (like warnings modal)

Reply:
fixed.

Reply:
For me, it's still not fixed. This is an example.

![Recording 2023-12-27 at 15 18 30](https://github.com/rango-exchange/rango-client/assets/99180934/bd0eb392-7aee-437d-9421-7d1ed73df1ab)

Reply:
LGTM. I've approved the PR. It would be great if we could improve the svg icon based on @Ikari-Shinji-re feedback.

Reply:
fixed.